### ISSUE TITLE
do not handle readstream error

### ISSUE THREAD

**Issue by FullStackZz** [devconcept/multer-gridfs-storage#205]
when readStream on error,there is not response on storage.fromStream.
i fund it, gridfs.js readstream not handle readStream on error
/_multer-gridfs-storage@4.2.0@multer-gridfs-storage/lib/gridfs.js
```
async fromStream(readStream, request, file) {
...
  writeStream.on('error', emitError);
}
```

**Comment by devconcept** [comment]
True but this creates a problem because if I reject the promise on the readable stream multer invokes error handling routines twice and this is more severe than not resolving the promise. I'll try to find a solution for this soon.

**Comment by devconcept** [comment]
Fixed in 5.0.0
